docs/source/queries/paged.md

```diff
@@ -119,6 +127,11 @@ let res2 = session
 # }
 ```
 
+> ***Warning***\
+> If the values are not empty, driver first needs to send a `PREPARE` request \
+> in order to fetch information required to serialize values. This will affect \
+> performance because 2 requests will be sent instead of 1.
```

Comment:
Suggestion: `s/because 2 requests will be sent/because 2 round trips will be required/`, to emphasize that those requests are executed serially (though technically speaking this is implied by the previous sentence).